cupy/statistics/histogram.py

```diff
@@ -40,7 +40,7 @@ def bincount(x, weights=None, minlength=None):
         raise ValueError('The weights and list don\'t have the same length.')
     if minlength is not None:
         minlength = int(minlength)
-        if minlength <= 0:
+        if minlength < 0:
```

Comment:
OK, this change. https://github.com/numpy/numpy/commit/ec6d4295a80e5df235d3f5445e6425581309c930